payments/utils.py

```diff
@@ -4,12 +4,13 @@
 import requests
```

Comment:
I know it hasnt been change in this issue, but I dont think we have this in requirements...